contracts/src/v0.8/llo-feeds/v0.4.0/test/verifier/VerifierVerifyTest.t.sol

```diff
@@ -0,0 +1,509 @@
+
+        changePrank(USER);
+        s_verifier.verify(signedReport, abi.encode(native), msg.sender);
+    }
+
+    function test_canVerifyNewerReportsWithNewerConfigs() public {
+        Signer[] memory signers = _getSigners(MAX_ORACLES);
+
+        uint8 MINIMAL_FAULT_TOLERANCE = 2;
+        BaseTest.Signer[] memory signersSubset1 = new BaseTest.Signer[](7);
+        signersSubset1[0] = signers[0];
+        signersSubset1[1] = signers[1];
+        signersSubset1[2] = signers[2];
+        signersSubset1[3] = signers[3];
+        signersSubset1[4] = signers[4];
+        signersSubset1[5] = signers[5];
+        signersSubset1[6] = signers[6];
+
+        address[] memory signersAddrSubset1 = _getSignerAddresses(signersSubset1);
+        // Config1
+        s_verifier.setConfig(signersAddrSubset1, MINIMAL_FAULT_TOLERANCE, new Common.AddressAndWeight[](0));
+
+        BaseTest.Signer[] memory signersSubset2 = new BaseTest.Signer[](7);
+        signersSubset2[0] = signers[7];
+        signersSubset2[1] = signers[8];
+        signersSubset2[2] = signers[9];
+        signersSubset2[3] = signers[10];
+        signersSubset2[4] = signers[11];
+        signersSubset2[5] = signers[12];
+        signersSubset2[6] = signers[13];
+
+        address[] memory signersAddrSubset2 = _getSignerAddresses(signersSubset2);
+        // Config2
+        s_verifier.setConfig(signersAddrSubset2, MINIMAL_FAULT_TOLERANCE, new Common.AddressAndWeight[](0));
+
+        BaseTest.Signer[] memory signersSubset3 = new BaseTest.Signer[](7);
+        signersSubset3[0] = signers[30];
+        signersSubset3[1] = signers[29];
+        signersSubset3[2] = signers[28];
+        signersSubset3[3] = signers[27];
+        signersSubset3[4] = signers[26];
+        signersSubset3[5] = signers[25];
+        signersSubset3[6] = signers[24];
+
+        address[] memory signersAddrSubset3 = _getSignerAddresses(signersSubset3);
+        // Config3
+        s_verifier.setConfig(signersAddrSubset3, MINIMAL_FAULT_TOLERANCE, new Common.AddressAndWeight[](0));
+
+        V3Report memory report = V3Report({
+            feedId: FEED_ID_V3,
+            observationsTimestamp: OBSERVATIONS_TIMESTAMP,
+            validFromTimestamp: uint32(block.timestamp),
+            nativeFee: uint192(DEFAULT_REPORT_NATIVE_FEE),
+            linkFee: uint192(DEFAULT_REPORT_LINK_FEE),
+            expiresAt: uint32(block.timestamp),
+            benchmarkPrice: MEDIAN,
+            bid: BID,
+            ask: ASK
+        });
+
+        BaseTest.Signer[] memory reportSigners = new BaseTest.Signer[](3);
+        reportSigners[0] = signers[30];
+        reportSigners[1] = signers[29];
+        reportSigners[2] = signers[28];
+
+        bytes memory signedReport = _generateV3EncodedBlob(report, s_reportContext, reportSigners);
+
+        s_verifierProxy.verify(signedReport, abi.encode(native));
+    }
+
+
+
+    /* uyncomment once we got ocnfirmation we need to support V1
```

Comment:
Ask Michael if V1 support is needed. If so then decoding the timestamp in verify has to address this possibility:

https://github.com/smartcontractkit/chainlink/pull/13813/files#diff-6758803ce1c961e4be4aaefaf9465ed647f6794412704a99a16a155ff3f40061R325-R329